src/core/DataFile.cpp

```diff
@@ -901,6 +901,38 @@ void DataFile::upgrade_1_1_91()
 }
 
 
+void DataFile::upgrade_1_2_0_rc2_23()
+{
+	// Upgrade to version 1.1.92
+	QDomNodeList list;
+	QDomElement el;
+	QDomNode node;
+
+	// syncmode for multiple tempo sync knobs
+	list = elementsByTagName( "syncmode" );
```

Comment:
So, this isn't working. The function I'm looking for would look something like:
list = elementThatHasAttribute( "syncmode" );